src/getHumanAge.test.js

```diff
@@ -3,8 +3,23 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
+  test('getHumanAge is a function', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('age of animals is less than 15', () => {
+    expect(getHumanAge(0, 14))
+      .toEqual([0, 0]);
+  });
+
+  test('age of animals is more or equal 15 and less than 24', () => {
+    expect(getHumanAge(15, 23))
+      .toEqual([1, 1]);
+  });
+
+  test('age of animals is more or equal 25', () => {
+    expect(getHumanAge(25, 50))
+      .toEqual([2, 7]);
+  });
```

Comment:
Recall test design techniques and write more tests, because these tests don't cover all cases of this function (it's not enough)

Reply:
I don't think that 5 test is enough to cover all cases for the next code:
![image](https://github.com/mate-academy/js_cat_and_dogs_years/assets/96179533/dd24701c-46b8-46d7-b8a3-49de4331429b)